pallets/ddc-staking/src/lib.rs

```diff
@@ -108,35 +117,47 @@ impl<AccountId, Balance: HasCompact + Copy + Saturating + AtLeast32BitUnsigned +
 				"filtering items from a bounded vec always leaves length less than bounds. qed",
 			);
 
-		Self { stash: self.stash, total, active: self.active, unlocking }
+		Self { stash: self.stash, total, active: self.active, chilling: self.chilling, unlocking }
+	}
+}
+
+/// Cluster staking parameters.
+#[derive(Clone, Decode, Encode, Eq, PartialEq, RuntimeDebugNoBound, TypeInfo)]
+#[scale_info(skip_type_params(T))]
+pub struct ClusterSettings<T: Config> {
```

Comment:
Do we assume that both storage and CDN nodes are in the same cluster?

Reply:
No assumption like that is required, but each cluster will have settings for both participant types.